PancakeSwap-Bots/Set-Masterchef-Settings/package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "set-masterchef-settings-bot",
+  "version": "0.0.1",
+  "description": "Detects function calls that modify settings in the PancakeSwap Masterchef contract",
+  "chainIds": [
+    56,
+    97
+  ],
+  "scripts": {
+    "build": "tsc",
+    "format": "prettier --write \"src/**/*.ts\"",
+    "start": "npm run start:dev",
+    "start:dev": "nodemon --watch src --watch forta.config.json -e js,ts,json  --exec \"npm run build && forta-agent run\"",
+    "start:prod": "forta-agent run --prod",
+    "tx": "npm run build && forta-agent run --tx",
+    "block": "npm run build && forta-agent run --block",
+    "range": "npm run build && forta-agent run --range",
+    "file": "npm run build && forta-agent run --file",
+    "publish": "forta-agent publish",
+    "info": "forta-agent info",
+    "logs": "forta-agent logs",
+    "push": "forta-agent push",
+    "disable": "forta-agent disable",
+    "enable": "forta-agent enable",
+    "keyfile": "forta-agent keyfile",
+    "test": "jest"
```

```suggestion
    "test": "jest --detectOpenHandles"
```
to prevent the warning while running `npm test`